Sources/LeafKit/LeafRenderer.swift

```diff
@@ -25,9 +25,12 @@ public final class LeafRenderer {
     public let sources: LeafSources
     /// The NIO `EventLoop` on which this instance of `LeafRenderer` will operate
     public let eventLoop: EventLoop
+    let _userInfo: SendableBox<[AnyHashable: Any]>
```

Comment:
This doesn't do what you think it does. The only way we can make `Any` `Sendable` is by tying this to an event loop

Reply:
Makes sense ... even with a `Value: Sendable` constraint it is still showing warnings.
Not sure what exactly to do though.
We need to change `Any` -> `any Sendable` as well as somehow conforming `AnyHashable` to `Sendable`.

Reply:
Though not sure what an event loop does that the locking mechanism can't ... I'm tempted to ask in the OpenSource slack.

Reply:
The documentation of `NIOLoopBound` does mention why...